shims/spark32/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcFileFormat.scala

```diff
@@ -90,19 +90,8 @@ class OrcFileFormat extends FileFormat with DataSourceRegister with Serializable
       options: Map[String, String],
       dataSchema: StructType): OutputWriterFactory = {
     val orcOptions = new OrcOptions(options, sparkSession.sessionState.conf)
-
-    val conf = job.getConfiguration
-
-    conf.set(MAPRED_OUTPUT_SCHEMA.getAttribute, OrcFileFormat.getQuotedSchemaString(dataSchema))
-
-    conf.set(COMPRESS.getAttribute, orcOptions.compressionCodec)
-
-    conf
-      .asInstanceOf[JobConf]
-      .setOutputFormat(classOf[org.apache.orc.mapred.OrcOutputFormat[OrcStruct]])
-
     if ("true" == sparkSession.sparkContext.getLocalProperty("isNativeApplicable")) {
```

Comment:
This might be subjective though it's a bit weird to see these Yoda conditions in our Scala code base. Perhaps we could invert them (in future patches).